docs/source/maintenance/add_peer.rst

```diff
@@ -0,0 +1,58 @@
+============
+Adding Peers
+============
+
+In HL Iroha, you can add new peers to the network while it is running.
+This is done by using a special command, `AddPeer <../develop/api/commands.html#add-peer>`_.
+
+Requirements
+============
+
+**There should be a peer that:**
+
+— runs with a Genesis Block (initial block of the blockchain) identical to the one on the peers already in the network;
+
+— has a resolvable address;
+
+— has a peer keypair (Ed25519 with SHA-3)
```

Comment:
Not necessarily Ed25519 with SHA-3, can be normal Ed25519 SHA-2 as well.

Reply:
Thanks, fixed!